src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -14,9 +15,15 @@
  */
 public class GATKVCFHeaderLines {
 
-    public static VCFInfoHeaderLine getInfoLine(final String id) { return infoLines.get(id); }
-    public static VCFFormatHeaderLine getFormatLine(final String id) { return formatLines.get(id); }
-    public static VCFFilterHeaderLine getFilterLine(final String id) { return filterLines.get(id); }
+    public static VCFInfoHeaderLine getInfoLine(final String id) {
+        return (infoLines.containsKey(id) ? infoLines.get(id) : new VCFInfoHeaderLine(id, 1, VCFHeaderLineType.Integer, "(Key not found -- type may be incorrect)"));
```

Comment:
This was the thing you helped me with a while ago with a totally perplexing HTSJDK header exception where the IntelliJ debugger doesn't output elements in a collection that are null, so the size and indexes look fine.  I wanted to put a placeholder because I never deal with the header minutiae until I'm ready to PR, but I can throw an exception.  I miss `--unsafe` mode.